### ISSUE TITLE
train each model on finalize data-record loss value 

### ISSUE THREAD

**Issue by mahshidghaffari** [mahshidghaffari/NER_GDPR_Extraction#21]
make sure you check the loss function per epochs
epochs = 3
